packages/ui-identicon/src/colors.js

```diff
@@ -23,8 +23,6 @@ module.exports = function colors (seeder: Seeder): ColorGen {
   return (alpha: number = 0.9): string => {
     const index = Math.floor(all.length * seeder());
 
-    return all.splice(index, 1)[0]
-      .alpha(alpha)
-      .string();
+    return String(all.splice(index, 1)[0].alpha(alpha));
```

Comment:
Looking at the color docs, I believe the correct approach should be something like -

```
// string hsl, i.e. will be something like 'hsl(320, 50%, 100%)'
return all.splice(index, 1)[0].alpha(alpha).hsl().string();
```

https://www.npmjs.com/package/color